### ISSUE TITLE
consider an FAQ or similar for common reticulate issues?

### ISSUE THREAD

**Issue by kevinushey** [rstudio/reticulate#264]
Somewhat prompted by https://github.com/rstudio/reticulate/issues/253#issuecomment-388616666, but it may be useful to collect various mechanisms for debugging `reticulate` or Python installation issues into a FAQ (perhaps as a GitHub wiki page).

It's tough to say how useful this could be since it seems like the matrix of issues that users encounter is vast enough that each issue a user encounters is unique to their specific environment, but it's worth thinking about.

(I'm mainly thinking about harvesting some of the solutions in the existing GitHub issue pages and collecting them into a single wiki / FAQ page)

**Comment by terrytangyuan** [comment]
Good idea! Also link them to https://stackoverflow.com/questions/tagged/reticulate to submit usage questions there.
